docs/source/usage.rst

```diff
@@ -0,0 +1,186 @@
+Using cool-seq-tool
+===================
+
+The core ``cool-seq-tool`` class provides three main methods: :ref:`transcript to genomic coordinates conversion <tr-to-g>`, :ref:`genomic to transcript coordinates conversion <g-to-tr>`, and :ref:`FASTA generation for a reference sequence <generate-fasta>`. This section provides an overview and describes basic procedures for each method.
+
+.. note::
+
+   Coordinate conversion methods are defined as ``async``, making use of Python's `coroutines library <https://docs.python.org/3/library/asyncio-task.html>`_. This means they must be ``await``\ ed. Users working in IPython or Jupyter should also see the `relevant documentation <https://ipython.readthedocs.io/en/stable/interactive/autoawait.html>`_ for more information.
+
+Initialization
+--------------
+
+The ``CoolSeqTool`` class can be initialized with no arguments to use default locations for all data sources:
+
+.. code-block:: python
+
+   from cool_seq_tool import CoolSeqTool
+   cst = CoolSeqTool()
+
+When used in an environment that contains existing Gene Normalizer and SeqRepo instances, those instances can be passed directly to the initialization method to reuse them:
+
+.. code-block:: python
+
+   from cool_seq_tool import CoolSeqTool
+   from cool_seq_tool.paths import SEQREPO_ROOT_DIR
+   from biocommons.seqrepo import SeqRepo
+   from gene.query import QueryHandler
+   from gene.database import create_db as create_gene_db
+
+   reusable_gene_normalizer = QueryHandler(create_gene_db())
+   reusable_seqrepo = SeqRepo(SEQREPO_ROOT_DIR)
+   cst = CoolSeqTool(
+       gene_query_handler=reusable_gene_normalizer,
+       sr=reusable_seqrepo
+   )
+
+See the :py:meth:`__init__ <cool_seq_tool.app.CoolSeqTool.__init__>` method documentation for more information and default argument values.
+
+.. _tr-to-g:
+
+Transcript to genomic coordinates
```

Comment:
I don't know if transcript is used for r. reference sequences as well, but it might be good to state that this is for c. (coding DNA) only atm